### PR TITLE
Translate SQL features

### PR DESCRIPTION
**Pull request by patiencedaur**
Part of https://github.com/tarantool/doc/issues/2496
Page: https://www.tarantool.io/ru/doc/latest/reference/reference_sql/sql-features/
Crowdin: https://crowdin.com/translate/tarantool-docs/84117/en-ru?filter=basic&value=2
Deployment: https://develop.tarantool.io/ru/doc/translate-sql-features/reference/reference_sql/sql-features/
Words: 2466

Refactor the table of SQL features to make it maintainable and translatable.

### CODE REVIEW

File: doc/reference/reference_sql/sql-features.rst
Comment (NickVolynkin) [inline doc/reference/reference_sql/sql-features.rst]: Please add trailing newline

Comment (patiencedaur) [reply doc/reference/reference_sql/sql-features.rst]: There is a trailing newline, its number is 1045.

File: doc/reference/reference_sql/sql-features.rst
Comment (igorkozloff) [inline doc/reference/reference_sql/sql-features.rst]:
```suggestion
            -   Expanded NULL predicate
```

Comment (alexandra-mara) [inline doc/reference/reference_sql/sql-features.rst]:
```suggestion
            -   Expanded NULL predicate
```

File: doc/reference/reference_sql/sql-features.rst
Comment (alexandra-mara) [inline doc/reference/reference_sql/sql-features.rst]: Hello @pgulutzan,
There's no example and no test results for E153, is that correct? This table contains tests results for all other items, even cursors, though Tarantool doesn't support them and we have no examples for them.

Comment (alexandra-mara) [inline doc/reference/reference_sql/sql-features.rst]: Hello @pgulutzan,
There's no example and no test results for E153, is that correct? This table contains tests results for all other items, even cursors, though Tarantool doesn't support them and we have no examples for them.

File: doc/reference/reference_sql/sql-features.rst
Comment (alexandra-mara) [inline doc/reference/reference_sql/sql-features.rst]: @pgulutzan, is this description correct? Shouldn't there be something like "Value expressions in ORDER BY clause"?

File: doc/reference/reference_sql/sql-features.rst
Comment (pgulutzan) [reply doc/reference/reference_sql/sql-features.rst]: It is not correct, I should have given an example.
I think UPDATE "view_containing_subquery" SET column1=0; would be good enough, and the result would be Fail.
I can add it in the English manual if nobody objects.
I am changing sql-features.rst anyway because of the style guide, see
https://github.com/tarantool/doc/commit/b63189cbf4dcc955f6b0f2bd4e14c4817811c26d

File: doc/reference/reference_sql/sql-features.rst
Comment (pgulutzan) [reply doc/reference/reference_sql/sql-features.rst]: @alexandra-mara: You are right, I erred, it should be "Value expressions in ORDER BY clause". Again, I can fix that in the English manual if there are no objections.

File: doc/reference/reference_sql/sql-features.rst
Comment (alexandra-mara) [reply doc/reference/reference_sql/sql-features.rst]: @pgulutzan this change should be performed in this PR then, as the table was refactored according to our style guides. If you can add this change to this PR, please do, if it's not convenient I can do it.
And can you please wait for us to merge this PR before finishing your work on gh-2516? This PR's almost ready, translation is completed, and I'm finishing the review

File: doc/reference/reference_sql/sql-features.rst
Comment (alexandra-mara) [reply doc/reference/reference_sql/sql-features.rst]: @pgulutzan If you can add this change to this PR, please do, if it's not convenient I can do it

File: doc/reference/reference_sql/sql-features.rst
Comment (pgulutzan) [reply doc/reference/reference_sql/sql-features.rst]: Whoops, I see now that we've both made the same change. Okay, you have priority because I must wait for a dev person to approve  all the other changes that I made in the SQL sections. I can cancel what I did in sql-features.rst, and you can make this change in "this PR" (i.e. 2537).

File: doc/reference/reference_sql/sql-features.rst
Comment (pgulutzan) [reply doc/reference/reference_sql/sql-features.rst]: @alexandra-mara: I believe it will be better to leave it to you, I am trying to avoid causing conflict. I see that someone has assigned me as a reviewer of this task, so please let me know when it is ready.

Comment (alexandra-mara) [reply doc/reference/reference_sql/sql-features.rst]: @pgulutzan that was an automatic assignment, as you commented on this PR you're considered its reviewer :)

File: doc/reference/reference_sql/sql-features.rst
Comment (alexandra-mara) [inline doc/reference/reference_sql/sql-features.rst]: @pgulutzan, since you're here, maybe you can help me out with this section, F311. It seems a little confusing that we first state that Tarantool doesn't have schemas, so we can't perform CREATE SCHEMA, but then we say that “Tarantool doesn't have CREATE TABLE inside CREATE SCHEMA”. If we can't perform CREATE SCHEMA, isn't it the main reason we can't comply with other items in F311 section?

Comment (pgulutzan) [reply doc/reference/reference_sql/sql-features.rst]: I don't see how that is confusing. Suppose someday we allow CREATE SCHEMA, but without any GRANT clauses within it. Then "Tarantool doesn't have GRANT inside CREATE SCHEMA" is still true. So CREATE SCHEMA is an essential prerequisite, but the specific thing that is missing is GRANT.

Comment (alexandra-mara) [reply doc/reference/reference_sql/sql-features.rst]: @pgulutzan okay, got it :)

**Review by alexandra-mara — APPROVED** [review]
Hello @pgulutzan, @patiencedaur
I made the changes we've discussed above, and I believe this PR is ready to merge

**Review by pgulutzan — APPROVED** [review]
Since I'm only marked as reviewer because of my comment, I only looked for the changes that we discussed.
I didn't see the change to E153 but see it is going to be done for issue #2659, and it will be marked Fail. So I initially thought that the count at the end, Total number of items marked “Fail”: 67, would need to be changed to 68. But now I counted the number of times that I see "Fail." or "Fail," not counting F021, it seems it would be 65. But I'm not good at counting. 
So I will say that it looks good to me.

**Comment by alexandra-mara** [comment]
@pgulutzan I've updated E153 with the example you provided via patch that was merged into this branch, you should be able to see it here https://github.com/tarantool/doc/pull/2537/commits/751b017fa305c4a2f4a20a2823b40508210394e4
Yes, of course we'll have to recount test results, I haven't thought of that. I actually searched through this page for "Fail" occurrences and there were 55, excluding those not from table and F021. And 80 "Okay"s. I was checking our current test deployment with these updates, it is not live yet

**Comment by pgulutzan** [comment]
@alexandra-mara: the number 55 surprised me, and now I have realized: all the privilege features are gone. This isn't right, because the list is supposed to be the same as the list in the standard. I suggest that I can put them back as part of the SQL changes for issue #2523. Also I might try changing the "widths:" to see whether it looks better if there is more space in the Example column.

**Comment by alexandra-mara** [comment]
@pgulutzan, I'm sorry I missed your message, I was away for a week. Yeah, surely we need to add privilege features, as part of #2523, or we can create a new issue

**Comment by pgulutzan** [comment]
@alexandra-mara: in my last commit for issue #2523 https://github.com/tarantool/doc/pull/2523/commits/ba284f0b86c9e677d66ce75754841618c6d9f10d  I copied your version of sql-features.rst and added the privilege features and removed a reference to "we" due to another change in the style guide. If I remember correctly, I didn't make other significant changes.

**Pull request merged**
